src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -472,17 +469,29 @@ const clickHandler = (graphContext, control, config, canvasSVG) => (
             shownTargets.push(item.key);
         }
     };
-    legendClickHandler(element);
+    const legend = legendClickHandler(element);
```

Comment:
I'd rename it to maybe `isLegendSelected` and flip the `if-else` blocks to make it more readable.

Reply:
Updated here https://github.com/cerner/carbon-graphs/pull/140/commits/e6eaf858e8e58510cc2e5b8f189e32df80b4059d.